### PR TITLE
Lz testing bullets

### PR DESCRIPTION
**Pull request by lzielezinska**

### CODE REVIEW

File: server/src/test/java/server/ABulletTest.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    }
+    
+    @Test
+    void testIsAndSetHostile() {
+        bulletExample1.setHostile(true);
+        boolean expected = bulletExample1.isHostile();
+        boolean actual = true;
+        assertThat(actual, is(equalTo(expected)));
+    }
+
```
Comment (iffyx) [inline server/src/test/java/server/ABulletTest.java]: USUN TO

File: server/src/test/java/server/GameControllerTest.java [hunk 2]
```diff
@@ -69,14 +78,16 @@ void checkBulletsData() {
...
 		JSONArray bulletsData = gameController.getBulletsData();
 		
-		assertFalse(bulletsData.isEmpty());
+		assertThat(bulletsData.isEmpty()).isFalse();
 	}
 	
 	@Test
 	void checkIfJSONArrayHasTheSameSizeLikeListOfBullets() {
-		assertEquals(getPreparedBullets().size(), gameController.getBulletsData().length());
+		assertThat(getPreparedBullets().size()).isEqualTo(gameController.getBulletsData().length());
```
Comment (Maniakrzelaza) [inline server/src/test/java/server/GameControllerTest.java]: The argument of _assertThat_ should be component that you are supposed to test, not prepared data. Please change order of the assertion.

File: server/src/test/java/server/GameControllerTest.java [hunk 3]
```diff
@@ -86,13 +97,15 @@ void checkOpponentsData() {
...
 		JSONArray opponentsData = gameController.getOpponentsData();
 		
-		assertFalse(opponentsData.isEmpty());
+		assertThat(opponentsData.isEmpty()).isFalse();
 	}
 	
 	@Test
-	void checkIfJSONArrayHasTheSameSizeLikeListOfPlayers() {
-		assertEquals(getPreparedPlayers().size(), gameController.getPlayersData().length());
+	void checkIfJSONArrayHasTheSameSizeLikeListOfOpponents() {
```
Comment (Maniakrzelaza) [inline server/src/test/java/server/GameControllerTest.java]: Same as above.

File: server/src/test/java/server/GameControllerTest.java [hunk 4]
```diff
@@ -102,14 +115,16 @@ void checkPlayersData() {
...
 		
-		assertFalse(playersData.isEmpty());
+		assertThat(playersData.isEmpty()).isFalse();
 	}
 	
 	@Test
-	void checkIfJSONArrayHasTheSameSizeLikeListOfOpponents() {
-		assertEquals(getPreparedBullets().size(), gameController.getBulletsData().length());
+	void checkIfJSONArrayHasTheSameSizeLikeListOfPlayers() {
+		assertThat(getPreparedPlayers().size()).isEqualTo(gameController.getPlayersData().length());
```
Comment (Maniakrzelaza) [inline server/src/test/java/server/GameControllerTest.java]: Same as above.

**Pull request closed without merging**